src/lib/wicked/adapter.js

```diff
@@ -136,9 +135,7 @@ class WickedAdapter {
      */
     removeConnection(connection) {
         return new Promise((resolve, reject) => {
-            const name = connection.name;
             this.deleteConnectionConfig(connection)
-                    .then(() => this.reloadConnection(name))
```

Comment:
So, we are not going to ifdown the interface when the configuration is removed?.

Reply:
Good question. I don't know, what do you think?

Reply:
Fixed in https://github.com/dgdavid/cockpit-wicked/pull/60. Now we have to do it explicitly.